src/components/button/button.js

```diff
@@ -72,7 +72,7 @@ module.exports = class Button extends Control {
     render() {
         return provide({
             block: this.bem.block,
-            attrs: {
+            attrs: Object.assign({
                 onMouseEnter: this._onMouseEnter.bind(this),
```

Comment:
We sent bindings to JXST. Base class doesn't to this. We can move common logic to Control, your are right.